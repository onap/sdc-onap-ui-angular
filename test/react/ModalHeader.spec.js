import React from 'react';
import ModalHeader from '../../src/react/ModalHeader.js';

import renderer from 'react-test-renderer';

describe('ModalHeader', () => {
	test('basic test', () => {
	    const header = renderer.create(<ModalHeader/>).toJSON();        
	    expect(header).toMatchSnapshot();
	});
});