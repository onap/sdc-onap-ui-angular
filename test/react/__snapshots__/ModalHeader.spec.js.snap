// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalHeader basic test 1`] = `
<div
  className="sdc-info__header sdc-modal__header"
>
  <div
    className="svg-icon-wrapper sdc-modal__icon sdc-modal__svg-use   bottom"
    disabled={undefined}
    onClick={undefined}
  >
    <svg />
    
  </div>
  <div
    className="svg-icon-wrapper sdc-modal__close-button-svg sdc-modal__close-button   bottom"
    disabled={undefined}
    onClick={undefined}
  >
    <svg />
    
  </div>
</div>
`;
